app/controllers/queues_controller.rb

```diff
@@ -12,4 +12,13 @@ def deduping
     criteria = @request.attributes.map { |col, val| " #{col} LIKE #{val}" unless val.nil? || val.empty? }.reject { |i| i.nil? || i.empty? }.join(" OR ")
     @similar_requests = RescueRequest.where(criteria)
   end
+
+  def dedupe_skip
+    RescueRequest.find(params.permit(:request_id)[:request_id]).review_tasks.create(user_id: @user.id, type: "dedupe", outcome: "skip")
+  end
+
+  def dedupe_done
+    RescueRequest.find(params.permit(:request_id)[:request_id]).review_tasks.create(user_id: @user.id, type: "dedupe", outcome: "done")
```

Comment:
You can just use `params[:request_id]` here - if you're not mass-assigning, you don't need strong params.